config/checker/com.intellij.astub

```diff
@@ -345,6 +345,8 @@ package com.intellij.openapi.vcs;
 
 @UI class CheckinProjectPanel {}
 
+// Experimentally verified that these methods can be called from outside UI thread.
+@AlwaysSafe
```

Comment:
Redundant, all methods are `@AlwaysSafe` by default, and `com.intellij.openapi.vcs` isn't marked as `@UIPackage`. Just added for clarity